### PR TITLE
tcl-tk: Build a bottle for Linuxbrew

### PR DESCRIPTION
**Pull request by rwhogg**

### CODE REVIEW

**Comment by rwhogg** [comment]
From https://travis-ci.org/Linuxbrew/homebrew-dupes/builds/218126688#L1549

```
/tmp/tcl-tk--tk-20170403-23206-17eox13/tk8.6.6/unix/../generic/tk.h:96:25: fatal error: X11/Xlib.h: No such file or directory
```

???

**Comment by sjackman** [comment]
```ruby
depends_on :x11 unless OS.mac?
```
?

**Comment by sjackman** [comment]
Strange, the default is `--without-x11`, so X11 should not be needed, and I prefer it that way.

**Comment by rwhogg** [comment]
Maybe it has something to do with:

```
 option "without-tk", "Don't build the Tk (window toolkit)"
 option "with-x11", "Build X11-based Tk instead of Aqua-based Tk"

  ...

 depends_on :x11 => :optional
```

Doesn't seem like it should have both an X11 requirement and an option.

File: tcl-tk.rb [hunk 1]
```diff
@@ -21,7 +22,6 @@ class TclTk < Formula
   option "with-threads", "Build with multithreading support"
   option "without-tcllib", "Don't build tcllib (utility modules)"
   option "without-tk", "Don't build the Tk (window toolkit)"
-  option "with-x11", "Build X11-based Tk instead of Aqua-based Tk"
```
Comment (rwhogg) [inline tcl-tk.rb]: There may be a very good reason that this is here. Will want to test this on Mac to make sure getting rid of it works as intended (and if so, submit this upstream once all the tap-migration business is done with).

Comment (sjackman) [inline tcl-tk.rb]: I believe the purpose of this option is only to change the text displayed by `brew options tcl-tk`

**Comment by rwhogg** [comment]
This repo is defunct, so closing this PR.

**Pull request closed without merging**